Tests/XcodeProjTests/Scheme/XCSchemeTests.swift

```diff
@@ -106,12 +106,34 @@ final class XCSchemeIntegrationTests: XCTestCase {
         XCTAssertEqual(subject.attributes["useTestSelectionWhitelist"], "YES")
         XCTAssertEqual(subject.attributes["testExecutionOrdering"], "random")
     }
+  
+  func test_write_testableReferenceAttributesValuesForSwiftTesting() {
```

Comment:
Let's also add a test for reading the `.swiftTestingOnly` case (i.e., when `parallelizable` is not specified)?

Reply:
Thanks for the prompt on the read tests. I've added one that is covered implicitly by the other reads, as the provided files do not have any.  I've also added cases on the writes for all 3 enum based inits + 2 bool init cases. Would it be worth creating another test file `XCSchemeTests+Parallelizable.swift` to cover all the cases here? I feel like I'm muddying the existing one.